src/index.html

```diff
@@ -7,12 +7,73 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image-container">
+        <img
+          src="./images/imac.jpeg"
+          alt="item card"
+          class="card__image"
+        />
+      </div>
+
+      <div class="card__name">
+        <div class="card__name-main">
+          APPLE A1419 iMac 27" Retina
```

Comment:
Avoid using `<br />` tags to manage layout and spacing. It's better to use CSS for these purposes. You can wrap each part of the title in a separate tag and manage the layout using CSS.

Reply:
+++, agree with AI
Why do you use `br` here?

Reply:
I believe you don't need br here at all